### DISCUSSION TITLE
Reusable Matrix "Partials" ?

### DISCUSSION THREAD

**Discussion by shantamg** [craftcms/cms#8722] [Ideas]
Pardon me if this idea is already documented somewhere, I did try searching.

I don't know about others, but I find that I'm often creating many matrix blocks with many of the exact same fields. I keep imagining a scenario where you could create "partials" to be reused across matrix fields. I know that might involve a pretty significant change to how those fields are organized, but perhaps there are even bigger ideas of how to make the whole situation more modular and flexible.

**Comment by jsunsawyer** [comment]
@shantamg We do this all the time.

You could set up a section for any number of re-usable content blocks. In your matrix field, you'd add a block type that contains an entry field so you can declare which re-usable content block to use. You'd just have to add a bit of template logic to load the correct content here.

Now your content is re-usable and a single update can affect all instances where your content is used.

For one of our clients, we just use entry fields to populate re-usable content blocks, ditching matrix completely. (This is mainly for translation reasons.)

**Comment by shantamg** [comment]
Oh of course, that's a great idea. Thanks!
